content/html-css/sprints/2/day-plan/index.md

```diff
@@ -7,8 +7,23 @@ weight = 3
 backlog= 'Module-HTML-CSS'
 backlog_filter= 'Week 2'
 [[blocks]]
+name="energiser"
+[[blocks]]
 name="Active listening"
 src="https://cyf-pd.netlify.app/blocks/active-listening/readme/"
+[[blocks]]
+name="morning-break"
+[[blocks]]
+name="Asking questions"
```

Comment:
The day plan view is saying the mermaid diagram contains a Syntax Error for the Asking Questions workshop: however it renders properly in the GIthub UI 🤔.
Happy for this PR to go through if we raise an issue to fix this in another PR - unless you can spot a quick fix now.